recipe/meta.yaml

```
@@ -32,19 +32,16 @@ outputs:
         - {{ pin_subpackage('gdma', max_pin='x.x') }}
     requirements:
       build:
-        - {{ stdlib("c") }}
+        - {{ stdlib("c") }}                                                      # [unix]
         - {{ compiler('c') }}                                                    # [unix]
         - {{ compiler('fortran') }}                                              # [unix]
+        - {{ stdlib("m2w64_c") }}                                                # [win]
         - {{ compiler('m2w64_c') }}                                              # [win]
         - {{ compiler('m2w64_fortran') }}                                        # [win]
         - cmake
         - ninja
-        - python 3.10                                                                      # only for generating a source file, so suppress all but one Py version. update as necessary.
-      run:
-        - m2w64-gcc-libgfortran                                                  # [win]
+        - python 3.12                                                                      # only for generating a source file, so suppress all but one Py version. update as necessary.
```

```suggestion
        - python
```
Instead of pinning python to 3.12, as I suggest above we can just skip building on other versions.